app/services/amr/data_feed_translator.rb

```diff
@@ -1,43 +1,54 @@
 module Amr
+  class DataFeedException < RuntimeError
```

Comment:
Duplicate of the new separate class file definition?

Reply:
Good point, fixed